arkouda/numeric.py

```diff
@@ -421,25 +421,30 @@ def cos(pda: pdarray) -> pdarray:
 
 
 @typechecked
-def hash(pda: pdarray, full: bool = True) -> Union[Tuple[pdarray, pdarray], pdarray]:
+def hash(
```

Comment:
should we include this in the `__all__` so people can do `ak.hash` instead of `ak.numeric.hash`?